src/app/components/views/homepage/homepage.component.html

```diff
@@ -1,6 +1,7 @@
 <ng-container *ngIf="weatherAlertResponse?.showAlertToast">
   <app-weather-alert [weatherAlert]="weatherAlertResponse"></app-weather-alert>
 </ng-container>
+<ng-container *ngIf="safetyDocumentUpdated"><app-newSafetyDocumentUpdate></app-newSafetyDocumentUpdate></ng-container>
```

Comment:
@looksliketyler should this be on multiple lines? Might be good to even if it isn't enforced practice. Nice use of atlas standards for directives! 

(This is where the component selector will need to be updated)